### ISSUE TITLE
Images do not open on click in the view

### ISSUE THREAD

**Issue by astrorobby** [flathub/org.kde.tellico#13]
Clicking on a link in the Entry View does not open the link, particularly the images. QDesktopServices::openUrl(QUrl) is used, so is it related to something with a portal, or needing to install another component?

Copied from https://bugs.kde.org/show_bug.cgi?id=467378

**Comment by tsdgeos** [comment]
Can you please be more clear what exactly are you doing?

You say "Clicking on a link", the bug you linked says "1. Click on a saved image in the entry view".

You're going to give us step by step instructions of what doesn't work.

**Comment by astrorobby** [comment]
> Can you please be more clear what exactly are you doing?
> 
> You say "Clicking on a link", the bug you linked says "1. Click on a saved image in the entry view".
> 
> You're going to give us step by step instructions of what doesn't work.

Right, sorry, I should have added more info when I transferred over the bug.

The saved images are links in the view, so clicking on them should open the default image reader, whatever is configured to open image/png files. More generically, if you have a URL field for an entry, clicking the link should open the default application for that link. Neither of those cases seem to work for me.

1. Open Tellico "flatpak run org.kde.tellico"
2. Click the Fields button in the tool bar and add a new URL field
  * Click New
  * Change Type from "Simple Text" to "URL", click OK to close dialog
3. Add a new entry with a url in that field
  * Click "Add Entry -> New Entry" in toolbar
  * Type "http://google.com" in the edit area for "New Field 1"
  * Click "Save Entry" and close the dialog
4. Select the entry in the window
5. Click on the google link

Expected behavior is that a browser opens up with that link. I'm just not sure what should happen if a browser has not been installed through flatpak or how that works. 

In the Tellico source code,  `QDesktopServices::openUrl(QUrl) ` is used, if that's relevant. Thanks for any help you might give.

**Comment by tsdgeos** [comment]
That works just fine for me.

I don't have a browser instaled trhough flatpak, my regular firefox is open.

Are you using Plasma as a desktop or something else?

**Comment by astrorobby** [comment]
Yes, using Plasma as my desktop. Thanks for checking on your side. I'm not sure what other info would be helpful to figure out why I don't get the expected response.

`:~>flatpak list`
`Name                          Application ID                            Version      Branch           Installation`
`Mesa                          org.freedesktop.Platform.GL.default       23.0.2       22.08            system`
`Mesa (Extra)                  org.freedesktop.Platform.GL.default       23.0.2       22.08-extra      system`
`Intel                         org.freedesktop.Platform.VAAPI.Intel                   22.08            system`
`openh264                      org.freedesktop.Platform.openh264         2.1.0        2.2.0            system`
`Breeze GTK theme              org.gtk.Gtk3theme.Breeze                  5.27.5       3.22             system`
`KDE Application Platform      org.kde.Platform                                       5.15-22.08       system`
`Tellico                       org.kde.tellico                           3.5          stable           system`

**Comment by tsdgeos** [comment]
Plasma X11 or Plasma Wayland? 

Do you have the plasma-integration package installed?

**Comment by astrorobby** [comment]
X11, running opensuse. I have their plasma5-integration-plugin package installed, yes.

**Comment by tsdgeos** [comment]
:shrug: no idea what can be different from your setup to mine :shrug:

**Comment by travier** [comment]
We haven't been able to reproduce that issue. Closing.
